packages/@vuepress/core/lib/node/build/index.js

```diff
@@ -77,9 +77,11 @@ module.exports = class Build extends EventEmitter {
     })
 
     // pre-render head tags from user config
+    // filter out meta tags for they will be injected in updateMeta.js
     this.userHeadTags = (this.context.siteConfig.head || [])
+      .filter(item => item[0] !== 'meta')
       .map(renderHeadTag)
-      .join('\n  ')
+      .join('\n    ')
```

Comment:
Why an extra space?

Reply:
Unify the indent (4 spaces) inside `<head>`

```html
<html>
  <head>
    <meta name="description" content="Vue-powered Static Site Generator">
    ...
  </head>
```